README.md

```diff
@@ -91,3 +91,8 @@ Questions & Comments
 --------------------
 
 For any and all feedback, please use the Issues on this repository. 
+
+License
+--------------------
+
+By downloading the .zip you are agreeing to the terms in the project [EULA](https://github.com/dotnet/cli/blob/master/packaging/osx/sharedframework/resources/en.lproj/eula.rtf).
```

Comment:
Is linking to the version in ```master``` correct/ok here or should I just leave it without a link?